src/Aspire.Dashboard/Otlp/WebApplicationExtensions.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using Aspire.Dashboard.Authentication;
+using Aspire.Dashboard.Otlp.Grpc;
+using Microsoft.AspNetCore.Mvc;
+using OpenTelemetry.Proto.Collector.Logs.V1;
+using OpenTelemetry.Proto.Collector.Metrics.V1;
+using OpenTelemetry.Proto.Collector.Trace.V1;
+
+namespace Aspire.Dashboard.Otlp;
+
+public static class WebApplicationExtensions
+{
+    // https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/large-object-heap
+    private const int MaxSizeLessThanLOH = 84999;
+
+    public static WebApplication ConfigureHttpOtlp(this WebApplication app, Uri httpEndpoint)
```

Comment:
I think, you have to configure CORS too, so browsers can send data to this endpoint.

Reply:
Lets avoid the cors configuration until we understand what it means to do this securely.